pkg/kubelet/kuberuntime/instrumented_services_test.go

```diff
@@ -40,7 +40,7 @@ func TestRecordOperation(t *testing.T) {
 
 	prometheusURL := "http://" + l.Addr().String() + "/metrics"
 	mux := http.NewServeMux()
-	//lint:ignore SA1019 ignore deprecated warning until we move off of global registries
+	//nolint:staticcheck // SA1019 ignore deprecated warning until we move off of global registries
```

Comment:
BTW, this is going to collide with another PR in flight, #106289

Reply:
@thockin that PR https://github.com/kubernetes/kubernetes/pull/106289 needs an approval to merge, can you approve and I'll rebase on top